backend/docker-compose.yaml

```diff
@@ -15,3 +15,53 @@ services:
     image: redis:4.0
     ports:
       - "6379:6379"
+  proddev-backend-wsgi:
+    depends_on:
+      - db
+      - redis
+    profiles:
+      - proddev
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8001:80"
+    environment:
+      - DATABASE_URL=postgres://postgres:postgres@db:5432/postgres
+      - DJANGO_SETTINGS_MODULE=officehoursqueue.settings.proddev
+    command: sh -c "python manage.py migrate && { DJANGO_SUPERUSER_PASSWORD=root python manage.py createsuperuser --no-input --username root --email root@pennlabs.org; /usr/local/bin/django-run; }"
+  proddev-backend-asgi:
+    depends_on:
+      - db
+      - redis
+    profiles:
+      - proddev
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8002:80"
+    environment:
+      - DATABASE_URL=postgres://postgres:postgres@db:5432/postgres
+      - DJANGO_SETTINGS_MODULE=officehoursqueue.settings.proddev
+    command: sh -c "python manage.py migrate && { DJANGO_SUPERUSER_PASSWORD=root python manage.py createsuperuser --no-input --username root --email root@pennlabs.org; /usr/local/bin/asgi-run; }"
```

Comment:
The createsuperuser command will fail if the database isn't ready yet. Instead, use a retry loop in the entrypoint

Reply:
The database will always be ready by the time that command runs right?